clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -2206,6 +2206,15 @@ float4 trunc(float4);
 // Wave* builtins
 //===----------------------------------------------------------------------===//
 
+/// \brief Returns true if the expression is true in any active lane in the
+/// current wave.
+///
+/// \param Val The boolean expression to evaluate.
+/// \return True if the expression is true in any lane.
+_HLSL_AVAILABILITY(shadermodel, 6.0)
```

Comment:
So this is correct.  And I can see the other wave intrinsics do this, but our default availability is 6.0. Is the thinking to do this that do the bookkeeping now so that it makes it easier to support pre 6.0 shader models i.e. FXC shaders at some point?

Reply:
I had propagated this for the sake of user documentation as well.

Reply:
I was just kind of following what others had done. But imo its worth keeping if for nothing else besides the documentation